Hunting Queries/OracleDatabaseAudit/OracleDBAuditDroppedTables.yaml

```diff
@@ -0,0 +1,25 @@
+id: e74b944a-c438-4949-b4d0-cdfe3a9354c9
+name: OracleDBAudit - Dropped Tables
+description: |
+  'Query searches for dropped tables.'
+requiredDataConnectors:
+  - connectorId: OracleDatabaseAudit
+    dataTypes:
+      - Syslog
+tactics:
+  - Impact
+query: |
+  let lbtime = 24h;
+  OracleDatabaseAuditEvent
+  | where TimeGenerated > ago(lbtime)
+  | where DbAction =~ 'DROP'
+  | extend TableName = replace(@'[,\(\)]', '', extract(@'(?i)SELECT(.*?)FROM\s(.*?)\s', 2, Action))
+  | where isnotempty(TableName)
+  | where TableName !in ('SELECT', 'select')

```

Comment:
use `| where TableName !~ 'select'` <-- case insensitive match

Reply:
updated

Reply:
Looks like this did not get updated.

Reply:
fixed